src/ex6/server/client/src/components/AddTask/AddTask.jsx

```diff
@@ -0,0 +1,121 @@
+import React, { useEffect, useState } from "react"
+import "./style.css"
+import { ItemClient } from "./item_client"
+import deleteIcon from "./images/delete_icon.svg"
+
+export const AddTask = () => {
+  const itemClient = new ItemClient()
+  const [task, setTask] = useState("")
+  const [tasks, setTasks] = useState([])
+  const [isLoading, setIsLoading] = useState(false)
+  const [error, setError] = useState(null)
+
+  useEffect(() => {
+    const fetchData = async () => {
+      setIsLoading(true)
+      try {
+        const result = await itemClient.getItems()
+        setTasks(result)
+      } catch (error) {
+        setError(error)
+      }
+      setIsLoading(false)
+    }
+    fetchData()
+  }, [])
+
+  console.log("tasks", tasks)
+  const handleSubmit = async (e) => {
```

Comment:
didn't manage to actually add an item
 every time I press the plus button - the page crashes
![image](https://user-images.githubusercontent.com/36815567/178155988-8190d16d-a07f-4e5f-9d36-dbeefdb44100.png)

then upon refresh 
![image](https://user-images.githubusercontent.com/36815567/178156022-c90d4f9e-abb4-462c-8bb0-9f96de8774f7.png)